docs/documentation/helm.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Helm
+description: Helm
+layout: docs
+permalink: /docs/helm
+---
+
+### Helm
+
+To let Dekorate to generate the Helm manifest files for a project, simply declare the following dependency part of your pom file:
+
+```xml
+<dependency>
+  <groupId>io.dekorate</groupId>
+  <artifactId>helm-annotations</artifactId>
+  <version>{{site.data.project.release.current-version}}</version>
+</dependency>
+```
+
+This dependency will generate the following Helm resources:
+- Chart.yaml
+- values.yaml
+- templates/*.yml the generated resources by Dekorate
+- <chart name>-<chart version>-helm.tar.gz
+
+#### Getting Started
+
+To generate the Helm resources, you first need to configure the Helm chart via properties:
```

Comment:
Is it mandatory to perform that in order to generate the helm resources ? I mean to create 2 properties part of the `properties.txt` file ? If yes, why don t you include the app version, ... as they exist with a Chart yaml file that we can have using `helm init` ?

Reply:
It's not mandatory. However, I see your point, will update this.

Reply:
Changed